src/configuration/types.rs

```diff
@@ -206,6 +206,22 @@ pub enum JsxQuoteStyle {
 
 generate_str_to_from![JsxQuoteStyle, [PreferDouble, "preferDouble"], [PreferSingle, "preferSingle"]];
 
+/// Whether to surround a JSX element or fragment with parentheses
+/// when it's the top JSX node and it spans multiple lines.
+#[derive(Clone, PartialEq, Copy, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub enum JsxMultiLineParensStyle {
```

Comment:
Little bit of bike shedding, but I think we can probably just name this `JsxMultiLineParens`. I'm going to merge this PR though, make that change and update deployment/schema.json in order to do a release.